Makefile

```
@@ -697,13 +696,33 @@ deploy-docs: venv
-# The demo script requires to install asciinema with: brew install asciinema to run on mac os envs.
-# Please ensure that all demos are named with the demo name and the suffix -demo-script.sh
-.PHONY: update-demos #EXHELP Validate demo recordings.
-update-demos:
-	@for script in hack/demo/*-demo-script.sh; do \
-	  nm=$$(basename $$script -script.sh); \
-	  ./hack/demo/generate-asciidemo.sh -n $$nm $$(basename $$script); \
+DEMO_OUTPUT_DIR ?= $(ROOT_DIR)/docs/demos
+
+.PHONY: update-demos
+update-demos: SOURCE_MANIFEST := $(EXPERIMENTAL_E2E_MANIFEST)
+update-demos: export MANIFEST := $(EXPERIMENTAL_RELEASE_MANIFEST)
```

checkmake (0.3.2)

[warning] Target "update-demos" defined multiple times (lines 702 and 703).

(uniquetargets)